pyk/src/integration_tests/kore/test_kore_client.py

```diff
@@ -38,17 +38,41 @@ class SimpleKoreClientTest(KoreClientTest):
     KOMPILE_MAIN_FILE = 'k-files/kore-rpc-test.k'
     KORE_MODULE_NAME = 'KORE-RPC-TEST'
 
-    TEST_DATA: Final[Tuple[Tuple[int, Mapping[str, Any], ExecuteResult], ...]] = (
-        (0, {}, BranchingResult(state=state(2), depth=2, next_states=(state(4), state(3)))),
-        (0, {'max_depth': 2}, DepthBoundResult(state=state(2), depth=2)),
-        (4, {}, StuckResult(state=state(6), depth=2)),
-    )
-
     def test_execute(self):
-        for i, (n, params, expected) in enumerate(self.TEST_DATA):
+        # Given
+        test_data: Final[Tuple[Tuple[int, Mapping[str, Any], ExecuteResult], ...]] = (
+            (0, {}, BranchingResult(state=state(2), depth=2, next_states=(state(4), state(3)))),
+            (0, {'max_depth': 2}, DepthBoundResult(state=state(2), depth=2)),
+            (4, {}, StuckResult(state=state(6), depth=2)),
+        )
+
+        for i, (n, params, expected) in enumerate(test_data):
```

Comment:
Let's name our tests, instead of numbers. I do think it makes it more readable.

Reply:
Good call, done.